### PR TITLE
implemented converyor belt controls

### PR DESCRIPTION
**Pull request by sarahfredrickson**
Added conveyor belt controls. Not sure if we still need drill cover or cup indices for the lazy susan.

### CODE REVIEW

File: science.py [hunk 1]
```diff
@@ -11,14 +11,14 @@
 N_SLOTS = 12
 DRILL_ARM_POWER = 0.5
 DRILL_POWER = 1.0
-
+CONVEYOR_BELT_POWER = 0x10000
```
Comment (abhaybd) [inline science.py]: Power values should be between -1 and 1, this is like ~65k

**Review by abhaybd — CHANGES_REQUESTED** [review]
Overall looks good, I think you made a typo in the power constant, and also can you clean up the extraneous whitespace changes?

In any case, after fixing the power constant this looks fine for preliminary testing

**Comment by sarahfredrickson** [comment]
Still debugging issues with the notifier for the sensor reading.

**Comment by abhaybd** [comment]
Science board completely changed, code has been cannibalized and used in the circ-24 branch

**Pull request closed without merging**